presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/json/format/MillisecondsSinceEpochFormatter.java

```diff
@@ -26,6 +26,8 @@
 {
     public static boolean isSupportedType(Type type)
     {
+        // milliseconds-since-epoch cannot encode a timezone hence writing TIMESTAMP WITH TIME ZONE
+        // is not supported to avoid losing the irrecoverable timezone information after write.
```

Comment:
i think it would be OK to write sans zone
https://github.com/prestosql/presto/blob/fbd4ca7f6ca1326a077fd06cd33b687cbfe17cf2/presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java#L612-L628

Reply:
See @losipiuk 's concerns at https://github.com/prestosql/presto/pull/5838#discussion_r519873801.

I can make the changes if we are sure we want to allow this.

Reply:
> See @losipiuk 's concerns at https://github.com/prestosql/presto/pull/5838#discussion_r519873801.

That's reasonable. And the contrary is not unreasonable either.

@losipiuk  please ACK the link above, the one to postgresql code.

Reply:
ACK. As discussed

Reply:
Can you add a TODO with an issue that we want to add support for writing here?